run-tests.ps1

```diff
@@ -30,21 +30,7 @@ function RunTest([string] $project, [string] $description,[bool] $skipBuild = $f
 
 
 $tests = @(
-  @{project = "..\tools\ExtensionsMetadataGenerator\test\ExtensionsMetadataGeneratorTests"; description="ExtensionsMetadataGenerator Tests"},
-  @{project ="WebJobs.Script.Tests"; description="Unit Tests"},
```

Comment:
undo

Reply:
Will do. Had just wanted to keep runs short while validating.